src/decorators/bootstrap/array.html

```diff
@@ -0,0 +1,32 @@
+<div  class="schema-form-array {{form.htmlClass}}"
```

Comment:
> as far as naming these files, can it be src/decorators/schema-form-bootstrap/array.html

I was trying to match the template path to make the files and originals easier to find (see index.ts), but I'm OK changing it too

```js
$templateCache.put('decorators/bootstrap/array.html', require('./decorators/bootstrap/array.html'));
```